benchmarks/coremark/Makefile

```diff
@@ -105,4 +105,8 @@ STACKSIZE += $(CONFIG_COREMARK_STACKSIZE)
 
 MAINSRC += core_main.c
 
+# Build with WebAssembly
+
+WASM_BUILD = y
```

Comment:
Does this need to be a config option?

Reply:
This macro just indicates that the application wants to be compiled as WASM, If CONFIG_INTERPRETERS_WAMR is not enabled, then this config has no effect